sys/net/gnrc/netif/init_devs/auto_init_at86rf215.c

```diff
@@ -19,8 +19,7 @@
 
 #ifdef MODULE_AT86RF215
 
-/* AT86RF215M only has a sub-GHz interface */
-#define USE_DUALBAND !(IS_USED(AT86RF215_DISABLE_SUBGHZ) || IS_USED(MODULE_AT86RF215M))
+#define USED_BANDS (IS_USED(MODULE_AT86RF215_SUBGHZ) + IS_USED(MODULE_AT86RF215_24GHZ))
```

Comment:
Can you add a compilation error if both are used but `GNRC_NETIF_NUMOF=1`?

Reply:
How about just disabling the sub-GHz interface instead if `GNRC_NETIF_NUMOF==1`?